frontend/server/src/Controllers/Course.php

```diff
@@ -32,6 +32,7 @@
  * @psalm-type StudentsProgressPayload=array{course: CourseDetails, students: list<CourseStudent>}
  * @psalm-type CourseProblem=array{accepted: int, alias: string, commit: string, difficulty: float, languages: string, letter: string, order: int, points: float, submissions: int, title: string, version: string, visibility: int, visits: int, runs: list<array{guid: string, language: string, source?: string, status: string, verdict: string, runtime: int, penalty: int, memory: int, score: float, contest_score: float|null, time: \OmegaUp\Timestamp, submit_delay: int}>}
  * @psalm-type IntroDetailsPayload=array{details: CourseDetails, progress?: AssignmentProgress, shouldShowFirstAssociatedIdentityRunWarning: bool}
+ * @psalm-type AddedProblem=array{alias: string, points: float|int}
```

Comment:
```suggestion
 * @psalm-type AddedProblem=array{alias: string, points: float}
```

como la intención es convertirlo a `float` eventualmente, mejor hay que no permitir `int` acá para hacer la conversión desde el principio.

Reply:
Listo